From e42f261111e760c025c7467d9e3446908f791456 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 2 Nov 2023 17:05:46 +0000
Subject: [PATCH] overlays: uart<n>-pi5: Add the pinctrl-0 property

Without the pinctrl-0 property in the overlays, the UARTs may not be
mapped correctly.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/uart0-pi5-overlay.dts | 1 +
 arch/arm/boot/dts/overlays/uart1-pi5-overlay.dts | 1 +
 arch/arm/boot/dts/overlays/uart2-pi5-overlay.dts | 1 +
 arch/arm/boot/dts/overlays/uart3-pi5-overlay.dts | 1 +
 arch/arm/boot/dts/overlays/uart4-pi5-overlay.dts | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/overlays/uart0-pi5-overlay.dts b/arch/arm/boot/dts/overlays/uart0-pi5-overlay.dts
index 62a2da68d39d..3cc9843b812d 100755
--- a/arch/arm/boot/dts/overlays/uart0-pi5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart0-pi5-overlay.dts
@@ -8,6 +8,7 @@ fragment@0 {
 		target = <&uart0>;
 		frag0: __overlay__ {
 			status = "okay";
+			pinctrl-0 = <&uart0_pins>;
 		};
 	};
 
diff --git a/arch/arm/boot/dts/overlays/uart1-pi5-overlay.dts b/arch/arm/boot/dts/overlays/uart1-pi5-overlay.dts
index 5e602eb199dc..739f5a941ffa 100755
--- a/arch/arm/boot/dts/overlays/uart1-pi5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart1-pi5-overlay.dts
@@ -8,6 +8,7 @@ fragment@0 {
 		target = <&uart1>;
 		frag0: __overlay__ {
 			status = "okay";
+			pinctrl-0 = <&uart1_pins>;
 		};
 	};
 
diff --git a/arch/arm/boot/dts/overlays/uart2-pi5-overlay.dts b/arch/arm/boot/dts/overlays/uart2-pi5-overlay.dts
index a7dd1937d0c2..1df956425d3a 100755
--- a/arch/arm/boot/dts/overlays/uart2-pi5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart2-pi5-overlay.dts
@@ -8,6 +8,7 @@ fragment@0 {
 		target = <&uart2>;
 		frag0: __overlay__ {
 			status = "okay";
+			pinctrl-0 = <&uart2_pins>;
 		};
 	};
 
diff --git a/arch/arm/boot/dts/overlays/uart3-pi5-overlay.dts b/arch/arm/boot/dts/overlays/uart3-pi5-overlay.dts
index 594661e3ab2a..d8ef51b403dd 100755
--- a/arch/arm/boot/dts/overlays/uart3-pi5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart3-pi5-overlay.dts
@@ -8,6 +8,7 @@ fragment@0 {
 		target = <&uart3>;
 		frag0: __overlay__ {
 			status = "okay";
+			pinctrl-0 = <&uart3_pins>;
 		};
 	};
 
diff --git a/arch/arm/boot/dts/overlays/uart4-pi5-overlay.dts b/arch/arm/boot/dts/overlays/uart4-pi5-overlay.dts
index 2abf97eddc77..7ce5be8cc95c 100755
--- a/arch/arm/boot/dts/overlays/uart4-pi5-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart4-pi5-overlay.dts
@@ -8,6 +8,7 @@ fragment@0 {
 		target = <&uart4>;
 		frag0: __overlay__ {
 			status = "okay";
+			pinctrl-0 = <&uart4_pins>;
 		};
 	};
 
-- 
2.34.1

